test/e2e/validation_test.go

```diff
@@ -667,14 +665,12 @@ func (tc *testContext) findNodeCSRs(nodeName string) ([]certificates.Certificate
 
 // validateUpgradeableCondition ensures that the operator's Upgradeable condition is correctly communicated to OLM
 func (tc *testContext) validateUpgradeableCondition(expected meta.ConditionStatus) error {
-	ocName, present := os.LookupEnv(condition.OperatorConditionName)
-	if !present {
-		// Implies operator is not OLM-managed
-		return nil
-	}
-	err := wait.Poll(retry.Interval, retry.ResourceChangeTimeout, func() (bool, error) {
-		oc := &operators.OperatorCondition{}
-		err := tc.client.Cache.Get(context.TODO(), types.NamespacedName{Namespace: tc.namespace, Name: ocName}, oc)
+	ocName, err := tc.getOperatorConditionName()
+	if err != nil || ocName == "" {
+		return errors.Errorf("unable to get operatorCondition name from namespace %s", tc.namespace)
```

Comment:
s/`tc.namespace`/`resourceNamespace`

To align with the value used in the API [call](https://github.com/openshift/windows-machine-config-operator/pull/1283/commits/f94484220f4198182a545297028b9fe10357c1c2#diff-19a6ecdd2ef106d645523b5e6ddc22fd6ddfd1608477cd0b0fbd85113c8c245dR691).

Reply:
We should use` tc.namespace` in all cases. The operator can technically be deployed in any namespace and it's best if we get this value from the test context rather than a constant.